exonum-java-binding/core/rust/integration_tests/tests/service_runtime_bootstrap.rs

```diff
@@ -35,19 +35,15 @@ fn bootstrap() {
         // Pass log4j path to avoid error messages of mis-configuration
         log_config_path: log4j_path(),
         port: 6300,
+        system_lib_path: None,
```

Comment:
This confirms the confusing nature of this parameter and its handling: `None` is passed, but that will make JavaServiceRuntime pick 'lib/native' under the hood
(it didn't use to do that).

I don't think though we shall ever skip 'java.library.path' — I don't know why it was made conditional (@vitvakatu :question: ).